packages/jaeger-ui/src/utils/tracking/conv-raven-to-ga.tsx

```diff
@@ -36,6 +37,16 @@ const FETCH_SYMBOLS = [
   { sym: '__IGNORE__', word: '', rx: /\.js(\.map)?$/i },
 ];
 
+interface SentryTransportOptions {
+  url: string;
+  data: any;
+  auth: {
+    sentry_version: string;
+    sentry_client: string;
+    sentry_key: string;
+  };
```

Comment:
why is this relevant to Jaeger? We're not submitting anything to Sentry.

Reply:
It was there in `RavenTransportOptions`, so I just had it here as well. Will remove.